replication_impl.go

```diff
@@ -47,3 +93,63 @@ func (c *client) DatabaseInventory(ctx context.Context, db Database) (DatabaseIn
 	}
 	return result, nil
 }
+
+// id of this batch
```

Comment:
Comment of go member must start with the name of the member.
So comment should be:
```
// BatchID returns the ID of this batch
```